src/ontology/cob.Makefile

```diff
@@ -36,88 +37,52 @@ products/:
 $(TMPDIR)/robot.jar: | $(TMPDIR)
 	if [ $(ROBOT_DOWNLOAD) = true ]; then curl -L -o $@ https://build.obolibrary.io/job/ontodev/job/robot/job/master/lastSuccessfulBuild/artifact/bin/robot.jar; fi
 
+
 ########################################
-# -- MAIN RELEASE PRODUCTS --
+# -- TEMPLATES --
 ########################################
 
-# build main release product
-REWIRE_PRECEDENCE = PR CHEBI
-cob.ttl: components/cob-to-external.tsv cob-native.owl
-	sssom rewire -I xml  -m $< $(patsubst %,--precedence %,$(REWIRE_PRECEDENCE)) cob-native.owl -o $@
-
-cob.owl: cob.ttl
-	robot merge --include-annotations true -i $< -i ontology-metadata.owl \
-	annotate --ontology-iri $(URIBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-	--output $@.tmp.owl && mv $@.tmp.owl $@
-.PRECIOUS: cob.owl
-
-cob-native.owl: $(SRC)
-	$(ROBOT) remove --input $< --select imports --trim false \
-		reason -r HERMIT \
-		annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		--output $@.tmp.owl && mv $@.tmp.owl $@
+$(TMPDIR)/cob-%.tsv: $(SCRIPTSDIR)/split-cob-edit.py cob-edit.tsv
+	$^ $(TMPDIR)
 
-# base file is main cob plus linking axioms
-#cob-base.owl: cob.owl $(COB_TO_EXTERNAL)
-#	$(ROBOT) merge $(patsubst %, -i %, $^) -o $@
-
-$(ONT)-base.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC)
-	$(ROBOT_RELEASE_IMPORT_MODE_BASE) \
-	$(SHARED_ROBOT_COMMANDS) \
-	annotate --link-annotation http://purl.org/dc/elements/1.1/type http://purl.obolibrary.org/obo/IAO_8000001 \
-		--ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		--output $@.tmp.owl && mv $@.tmp.owl $@
-
-cob-base-reasoned.owl: cob-base.owl
-	$(ROBOT) remove --input $< --select imports --trim false \
-		reason -r HERMIT \
-		annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		--output $@.tmp.owl && mv $@.tmp.owl $@
-
-cob-examples-reasoned.owl: cob-base.owl $(COB_EXAMPLES)
-	$(ROBOT) remove --input $< --select imports --trim false \
-		merge $(patsubst %, -i %, $^) \
-		reason -r HERMIT \
-		annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		--output $@.tmp.owl && mv $@.tmp.owl $@
-
-# TSV export (may depend on dev version of robot export)
-cob.tsv: cob.owl
```

Comment:
cob.tsv was I believe an not entirely useless output - any reason it was dropped?

Reply:
No particular reason.

Reply:
Restored 4bb8a417db0b2e86737b738f60f4df8f2d53303f